docs/topics/commonly_used_plugins.rst

```diff
@@ -10,284 +10,69 @@ Some commonly-used plugins
-.. _django filer CMS plugin: https://github.com/stefanfoulis/cmsplugin-filer
-
-****
-Text
-****
-
-Consider using `djangocms-text-ckeditor
-<https://github.com/divio/djangocms-text-ckeditor>`_ for displaying text. You
-may of course use your preferred editor; others are available.
-
-..  module:: djangocms_video
-
-..  class:: djangocms_video.cms_plugins.VideoPlugin
-
-*****
-Video
-*****
-
-Available on `GitHub (divio/djangocms-video) <http://github.com/divio/djangocms-video>`_
-and on `PyPi (djangocms-video) <https://pypi.python.org/pypi/djangocms-video>`_.
-
-Plays Video Files or YouTube / Vimeo Videos. Uses the `OSFlashVideoPlayer
-<http://github.com/FlashJunior/OSFlashVideoPlayer>`_. When uploading videos use either
-``.flv`` files or H264 encoded video files.
-
-Please install it using ``pip`` or similar and be sure you have the following in your project's
-:setting:`django:INSTALLED_APPS` setting::
-
-    INSTALLED_APPS = (
-        # ...
-        'djangocms_video',
-        # ...
-    )
-
-There are some settings you can set in your ``settings.py`` to overwrite some
-default behaviour:
-
-* ``VIDEO_AUTOPLAY`` ((default: ``False``)
-* ``VIDEO_AUTOHIDE`` (default: ``False``)
-* ``VIDEO_FULLSCREEN`` (default: ``True``)
-* ``VIDEO_LOOP`` (default: ``False``)
-* ``VIDEO_AUTOPLAY`` (default: ``False``)
-* ``VIDEO_BG_COLOR`` (default: ``"000000"``)
-* ``VIDEO_TEXT_COLOR`` (default: ``"FFFFFF"``)
-* ``VIDEO_SEEKBAR_COLOR`` (default: ``"13ABEC"``)
-* ``VIDEO_SEEKBARBG_COLOR`` (default: ``"333333"``)
-* ``VIDEO_LOADINGBAR_COLOR`` (default: ``"828282"``)
-* ``VIDEO_BUTTON_OUT_COLOR`` (default: ``"333333"``)
-* ``VIDEO_BUTTON_OVER_COLOR`` (default: ``"000000"``)
-* ``VIDEO_BUTTON_HIGHLIGHT_COLOR`` (default: ``"FFFFFF"``)
-
-You should take care that the directory defined by the configuration setting
-:setting:`CMS_PAGE_MEDIA_PATH` (by default ``cms_page_media/`` relative to
-:setting:`django:MEDIA_ROOT`) is writeable by the user under which django will be
-running.
-
-.. note:: For more advanced use cases where you would like to upload your media
-          to a central location, consider using  `django-filer`_ with
-          `django filer CMS plugin`_ and its ``cmsplugin_filer_video`` component
-          instead.
-
-.. _django-filer: https://github.com/stefanfoulis/django-filer
-.. _django filer CMS plugin: https://github.com/stefanfoulis/cmsplugin-filer
-
-..  module:: djangocms_twitter
-
-..  class:: djangocms_twitter.cms_plugins.TwitterRecentEntriesPlugin
-
-..  class:: djangocms_twitter.cms_plugins.TwitterSearchPlugin
-
-*******
-Twitter
-*******
-
-We recommend one of the following plugins:
-
-* https://github.com/nephila/djangocms_twitter
-* https://github.com/changer/cmsplugin-twitter
-
-.. warning:: These plugins are not currently compatible with Django 1.7.
-
-..  module:: djangocms_inherit
+These packages are no longer officially guaranteed support by the django CMS project, but they have good community support.
```

Comment:
I would say "These packages are no longer actively maintained by the django CMS core team, but they have good community support."